### ISSUE TITLE
Add "Website" page

### ISSUE THREAD

**Issue by bnvk** [TheTorProject/ooni-explorer#31]
Domain names which have cases of censorship often have multiple tests run. It makes sense to have a dedicated page (similar to the "country" page) for websites, something like 

```
https://api.ooni.io/website/www.crazyshit.com
```

On this page would be content such as:

- [ ] Some metadata about a given website
- [ ] Alexa ranking, # of Google results "www.crazyshit.com"
- [x] A list of all reports run to date on given website
- [x] Categories
- [x] Countries where censored / blocked / etc

**Comment by simonv3** [comment]
Related - when listing these domains in the country censorship view it'd be great to show what kind of website they are, but at the same time do we want to predetermine that kind of stuff?

**Comment by hellais** [comment]
@simonv3 yes we could actually already begin doing that since there is a database table that includes the the categories of websites. I would however suggest we first take care of all the "low hanging fruit" and then proceed to implementing that.

**Comment by simonv3** [comment]
I can tackle this. @hellais, is there a reverse join / relationship on the database that gives us the countries a domain is blocked in? I can tinker with the data to figure it out, but I thought it might be faster for you to draft it, and then I can implement the endpoint and the front-end for it.

**Comment by hellais** [comment]
@simonv3 this information is stored inside of the materialised view `blockpage_urls`. Anything that is contained inside of that view is to be considered as blocked. I think it would be interesting however to join this with also measurements that show when the site in questions appears to not be blocked, hence doing the ^-1 of this big query:

```
 SELECT metrics.input,
    metrics.report_id,
    metrics.test_start_time,
    metrics.probe_cc,
    metrics.probe_asn
   FROM metrics
  WHERE metrics.probe_cc = 'RU'::text AND metrics.test_name = 'http_requests'::text AND (metrics.test_keys ->> 'body_length_match'::text) = 'false'::text AND (((((metrics.test_keys -> 'requests'::text) -> 0) -> 'response'::text) ->> 'body'::text) ~~ '%http://eais.rkn.gov.ru/%'::text OR ((((metrics.test_keys -> 'requests'::text) -> 1) -> 'response'::text) ->> 'body'::text) ~~ '%http://eais.rkn.gov.ru/%'::text)
UNION
 SELECT metrics.input,
    metrics.report_id,
    metrics.test_start_time,
    metrics.probe_cc,
    metrics.probe_asn
   FROM metrics
  WHERE metrics.probe_cc = 'IR'::text AND metrics.test_name = 'http_requests'::text AND (metrics.test_keys ->> 'body_length_match'::text) = 'false'::text AND (((((metrics.test_keys -> 'requests'::text) -> 0) -> 'response'::text) ->> 'body'::text) ~~ '%iframe src="http://10.10%'::text OR ((((metrics.test_keys -> 'requests'::text) -> 1) -> 'response'::text) ->> 'body'::text) ~~ '%iframe src="http://10.10%'::text)
UNION
 SELECT metrics.input,
    metrics.report_id,
    metrics.test_start_time,
    metrics.probe_cc,
    metrics.probe_asn
   FROM metrics
  WHERE metrics.probe_cc = 'TR'::text AND metrics.test_name = 'http_requests'::text AND (metrics.test_keys ->> 'body_length_match'::text) = 'false'::text AND (((((metrics.test_keys -> 'requests'::text) -> 0) -> 'response'::text) ->> 'body'::text) ~~ '%uyarınca yapılan teknik inceleme ve hukuki değerlendirme sonucunda bu internet%'::text OR ((((metrics.test_keys -> 'requests'::text) -> 1) -> 'response'::text) ->> 'body'::text) ~~ '%uyarınca yapılan teknik inceleme ve hukuki değerlendirme sonucunda bu internet%'::text)
UNION
 SELECT metrics.input,
    metrics.report_id,
    metrics.test_start_time,
    metrics.probe_cc,
    metrics.probe_asn
   FROM metrics
  WHERE metrics.probe_cc = 'GR'::text AND metrics.test_name = 'http_requests'::text AND (metrics.test_keys ->> 'body_length_match'::text) = 'false'::text AND (((((metrics.test_keys -> 'requests'::text) -> 0) -> 'response'::text) ->> 'body'::text) ~~ '%www.gamingcommission.gov.gr/index.php/forbidden-access-black-list/%'::text OR ((((metrics.test_keys -> 'requests'::text) -> 1) -> 'response'::text) ->> 'body'::text) ~~ '%www.gamingcommission.gov.gr/index.php/forbidden-access-black-list/%'::text)
UNION
 SELECT metrics.input,
    metrics.report_id,
    metrics.test_start_time,
    metrics.probe_cc,
    metrics.probe_asn
   FROM metrics
  WHERE metrics.probe_cc = 'SA'::text AND metrics.test_name = 'http_requests'::text AND (((((metrics.test_keys -> 'requests'::text) -> 0) -> 'response'::text) -> 'headers'::text) ->> 'Server'::text) ~~ 'Protected by WireFilter%'::text
UNION
 SELECT metrics.input,
    metrics.report_id,
    metrics.test_start_time,
    metrics.probe_cc,
    metrics.probe_asn
   FROM metrics
  WHERE metrics.probe_cc = 'ID'::text AND metrics.test_name = 'http_requests'::text AND (((((metrics.test_keys -> 'requests'::text) -> 0) -> 'response'::text) -> 'headers'::text) ->> 'Location'::text) ~~ 'http://internet-positif.org%'::text
UNION
 SELECT metrics.input,
    metrics.report_id,
    metrics.test_start_time,
    metrics.probe_cc,
    metrics.probe_asn
   FROM metrics
  WHERE metrics.probe_cc = 'CN'::text AND metrics.test_name = 'http_requests'::text AND (metrics.test_keys -> 'experiment_failure'::text) <> 'null'::jsonb AND (metrics.test_keys -> 'control_failure'::text) = 'null'::jsonb;
```

**Comment by hellais** [comment]
I think we can consider this implemented for the purpose of the 0.1 beta release.
